neuralforecast/models/nbeats/nbeats.py

```diff
@@ -797,3 +797,121 @@ def forecast(self: NBEATS, Y_df: pd.DataFrame, X_df: pd.DataFrame = None, S_df:
     forecast_df['y'] = forecast.flatten()
 
     return forecast_df
+
+
+# Cell
+@patch
+def predict(self: NBEATS, Y_df: pd.DataFrame, X_df: pd.DataFrame = None, S_df: pd.DataFrame = None,
```

Comment:
Probably `predict_in_sample` would be a better name? I think the `predict` method should be the same as the `forecast` method if we want to include it.

Reply:
lets use fit predict, its more common